src/docs/education-and-use-cases/how-to-spot-tops-with-price---network-activity-divergences/index.md

```diff
@@ -13,18 +13,16 @@ If these indicators spike during a pump - signalling a rise in speculative inter
 
 Take ENJ for example, which had a giant spike in active addresses at the beginning of December, coinciding with the price rally:
 
-![](image1.png)
+![enj-active-addresses-and-network-growth](image1.png)
```

Comment:
I just swapped the names of the images themselves, so they appear in the proper order in the markdown file